dev/RadioMenuFlyoutItem/RadioMenuFlyoutItem_rs2_themeresources.xaml

```diff
@@ -2,103 +2,89 @@
 <ResourceDictionary
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:contract7Present="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract,7)"
     xmlns:local="using:Microsoft.UI.Xaml.Controls">
 
     <Style TargetType="local:RadioMenuFlyoutItem" BasedOn="{StaticResource DefaultRadioMenuFlyoutItemStyle}" />
 
-    <!-- Same as default rs2 style for ToggleMenuFlyoutItem -->
+    <!-- Same as rs5 DefaultToggleMenuFlyoutItemStyle -->
     <Style x:Key="DefaultRadioMenuFlyoutItemStyle" TargetType="local:RadioMenuFlyoutItem">
-        <Setter Property="Background" Value="{ThemeResource ToggleMenuFlyoutItemBackground}" />
```

Comment:
Changes to rs2,4,5 is more than what i would have expected. are these intentional changes ?